### ISSUE TITLE
FT.EXPLAIN returns empty explanation for vector query

### ISSUE THREAD

**Issue by oshadmi** [RediSearch/RediSearch#2472]
Explanation should not be empty, for example, on master (04f27b0f):
```
127.0.0.1:6379> FT.CREATE idx SCHEMA v VECTOR HNSW 6 TYPE FLOAT32 DIM 2 DISTANCE_METRIC L2
OK
127.0.0.1:6379> FT.EXPLAIN idx '@v:[$vec topk 4]' PARAMS 2 vec aaaaaaaa
"VECTOR { }\n"
127.0.0.1:6379> 

```

**Comment by GuyAv46** [comment]
This is fixed in feature_vecsim_search_API_update branch

**Comment by oshadmi** [comment]
Fixed by #2498
